browserexport/browsers/all.py

```diff
@@ -1,15 +1,16 @@
 from typing import List, Type
-from .common import Browser
-from .firefox import Firefox
-from .waterfox import Waterfox
-from .palemoon import Palemoon
-from .chrome import Chrome
-from .chromium import Chromium
-from .brave import Brave
-from .safari import Safari
-from .vivaldi import Vivaldi
-from .firefox_mobile import FirefoxMobile
-from .firefox_mobile_legacy import FirefoxMobileLegacy
+from browserexport.browsers.common import Browser
+from browserexport.browsers.firefox import Firefox
+from browserexport.browsers.waterfox import Waterfox
+from browserexport.browsers.palemoon import Palemoon
```

Comment:
whoops. this seems to have happened automatically from the formatter, want me to revert?

Reply:
Yeah, should be relative imports inside the package

Reply:
done, i think my editor defaults to using black to format, which seems incompatible with setup.cfg. 

for new projects, are you using the pyproject.toml thing they recommend?

Reply:
I use black as well, should work [fine with setup.cfg](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html) as long as its running the formatter from the root dir -- I don't have anything special in the setup.cfg for black though, is all just the defaults

Reply:
Ah nevermind, I misread, it does only use pyproject.toml

But not using it anyways, so must be something else changing the imports to absolute, unless you have a global black configuration file

Reply:
weird yeah no idea, i don't have a config, the only python i write is in colab notebooks usually.

thanks, i'll test this and propose changes if it doesn't work